test/README.md

```diff
@@ -7,13 +7,23 @@ You can find instructions on how to run this test suite on the Falco website [he
 ## Test suites
 
 - [falco_tests](./falco_tests.yaml)
-- [falco_traces](./falco_traces.yaml)
+- [falco_traces](./falco_traces.yaml.in)
 - [falco_tests_package](./falco_tests_package.yaml)
 - [falco_k8s_audit_tests](./falco_k8s_audit_tests.yaml)
 - [falco_tests_psp](./falco_tests_psp.yaml)
 
 ## Running locally
 
+This step assumes you already built Falco.
```

Comment:
> Note that the test suites might not work with a [Debug buid](https://falco.org/docs/source/#specify-the-build-type), please use a built with type `Release` (eg. `cmake -DCMAKE_BUILD_TYPE=Release ..`)

As I have found out, the debug build has several unresolved issues that broke some integration tests. To avoid headaches, I believe it worths adding the above note.